cuegui/cuegui/Constants.py

```diff
@@ -133,10 +127,13 @@ def __packaged_version():
 LOGGER_FORMAT = __config.get('logger.format')
 LOGGER_LEVEL = __config.get('logger.level')
 
-EMAIL_SUBJECT_PREFIX = __config.get('email.subject_prefix')
-EMAIL_BODY_PREFIX = __config.get('email.body_prefix')
-EMAIL_BODY_SUFFIX = __config.get('email.body_suffix')
-EMAIL_DOMAIN = __config.get('email.domain')
+EMAIL_SUBJECT_PREFIX = __config.get('email.subject_prefix', "cuemail: please check ")
+EMAIL_BODY_PREFIX = __config.get('email.body_prefix', "Your PSTs request that you check:\n")
```

Comment:
At Sony Pictures Imageworks, 'PST' stands for Production Services Technician. However, not all companies using OpenCue may be familiar with or use this specific terminology. I recommend adopting a more generic term, such as 'team members' or 'support team' or another option to ensure broader understanding and applicability across different environments.

Consider replacing all the PSTs or PST in all the files of this PR with a more generic term, such as 'team members' or 'support team' or another option.

Reply:
```suggestion
EMAIL_BODY_PREFIX = __config.get('email.body_prefix', "Your Support Team requests that you check:\n")
```